exporter/ststopologyexporter/internal/topology_test.go

```diff
@@ -527,7 +527,7 @@ func TestTopology_addKubernetes(t *testing.T) {
 			SourceId:   "urn:opentelemetry:kubernetes:/ye-cluster:ye-k8s-namespace:pod/ye-pod-name",
 			TargetId:   "urn:opentelemetry:namespace/ns:service/ye-service:serviceInstance/ye-service",
 			Type: RelationType{
-				Name: "kubernetes to otel",
+				Name: "kubernetes-to-otel",
```

Comment:
Do this change of names will have some influence? Is this something that needs to be updated on the stackpack?

Reply:
the change itself was already in for some time - but the tests hadn't been updated.